examples/server/tests/features/security.feature

```diff
@@ -47,5 +47,5 @@ Feature: Security
       | localhost       | Access-Control-Allow-Origin      | localhost         |
       | web.mydomain.fr | Access-Control-Allow-Origin      | web.mydomain.fr   |
       | origin          | Access-Control-Allow-Credentials | true              |
-      | web.mydomain.fr | Access-Control-Allow-Methods     | POST              |
+      | web.mydomain.fr | Access-Control-Allow-Methods     | *                 |
```

Comment:
Actually can we just restrict to GET and POST ?

Reply:
I assume we want GET, POST, OPTIONS so that preflight requests are allowed.